### PR TITLE
add reminders

### PR DESCRIPTION
**Pull request by vhouchouas**

### CODE REVIEW

File: slackEvents.php [hunk 1]
```diff
@@ -215,5 +215,21 @@ function register($url, $userid, $in, $request) {
...
+        $datetime->modify("-1 day");
+        
+        if($in) {
+            $summary = (string)$vevent->SUMMARY;
+            $response = $this->api->reminders_add($userid, "Rappel pour l'événement: $summary", $datetime);
+            $this->log->debug("reminder created ({$response->reminder->id})");
+        } else {
+            $reminders = $this->api->reminders_list();
+            $reminder_id = getReminderID($reminders["reminders"], $userid, $datetime);
+            $this->api->reminders_delete($reminder_id);
```
Comment (gturri) [inline slackEvents.php:231]: Should we check whether `! is_null($reminder_id)` before calling this?
(IIUC, in the general case, if we run this it means that the user was already registered on the event so we should expect a reminder to exist, but I guess there can be corner cases where this assumption is not verified (for instance a user manually added directly to an event through the calendar UI by an admin))

File: slackEvents.php [hunk 2]
```diff
@@ -215,5 +215,21 @@ function register($url, $userid, $in, $request) {
         $this->log->debug("register mail $profile->email $profile->first_name $profile->last_name");
         $r = $this->agenda->updateAttendee($url, $profile->email, $in, $profile->first_name . ' ' . $profile->last_name);
         $this->app_home_page($userid, $request);
-    }    
+
+        $vevent = $this->agenda->getEvent($url)->VEVENT;
+        $datetime = $vevent->DTSTART->getDateTime();
+        $datetime->setTime(19, 0, 0, 0);
```
Comment (gturri) [inline slackEvents.php:221]: as we discussed, don't set 19h to avoid conflicts

**Review by gturri — CHANGES_REQUESTED** [review]

File: slackEvents.php (hunk 2 above)
Comment (vhouchouas) [reply slackEvents.php]: Fixed in d8aad75

File: slackEvents.php (hunk 1 above)
Comment (vhouchouas) [reply slackEvents.php]: Indeed, fixed in 81ea535

**Pull request merged**